test/sdn_migration/scripts/validate-sdn-ovn-migration-jobs.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set +e
+
+UNIQUE_ID=$(date +%s%N) # Generate a unique identifier for this instance
+CONTAINER_NAME="sdn-ovn-migration-${UNIQUE_ID}" # Name of the container based on the unique identifier
+
+# Check if the container already exists
+if podman ps -a --format '{{.Names}}' | grep -q "^${CONTAINER_NAME}$"; then
+    echo "Container ${CONTAINER_NAME} already exists. Exiting."
+    exit 1
+fi
```

Comment:
`UNIQUE_ID` will be different every time it is called, so this container will never exist under this name

```
❯ date +%s%N
1727165426147099231

❯ date +%s%N
1727165426791158471

❯ date +%s%N
1727165427253005030

❯ date +%s%N
1727165427649098720
```

Reply:
cc @ritmun @MrSantamaria ^